src/pwr.ps1

```diff
@@ -727,8 +727,25 @@ if (-not $Run) {
 	}
 }
 
+function Get-PwrConfig {
+	$PwrCfgDir = $ExecutionContext.SessionState.Path.CurrentLocation
+	do {
+		$PwrCfg = "$PwrCfgDir\pwr.json"
+		if (Test-Path -Path $PwrCfg -PathType Leaf) {
+			try {
+				return Get-Content $PwrCfg -ErrorAction 'SilentlyContinue' | ConvertFrom-Json
+			} catch {
+				Write-PwrFatal "pwr: bad JSON parse of file $PwrCfg - $_"
+			}
+		}
+		$PwrCfgDir = Split-Path -Parent $PwrCfgDir
+	} while ($PwrCfgDir -ne $null)
+}
 
-$PwrConfig = Get-Content 'pwr.json' -ErrorAction 'SilentlyContinue' | ConvertFrom-Json
+$PwrConfig = Get-PwrConfig
+if ($PwrConfig) {
+	Write-Debug "pwr: using config at $PwrCfg"
+}
```

Comment:
Let's move this to `Assert-NonEmptyPwrPackages`

Otherwise, we are doing recursive directory all the time when it might not be needed.